### ISSUE TITLE
How to encourage discussions on PRs in GraphQL Spec repo?

### ISSUE THREAD

**Issue by IvanGoncharov** [graphql/graphql-wg#114]
Just as a continuation on disscusion that we had during WG.

A few actionable ideas so far are:

- [ ] More descriptive tags on issues and PRs 
- [ ] Have a "This week in GraphQL specification" newsletter

Would be great to hear other ideas?

**Comment by craigsmitham** [comment]
@IvanGoncharov first - really appreciate your past help and willingness to discuss issues here and in GraphQL JS. So, more ways we can encourage good discussion will be great.

I think the Ember community has done this well. They have a blog/newsletter published weekly with a roundup of information relevant to the core Ember framework and libraries with links to their forum and GH issues for more discussion. They also just switched from Slack to Discord for real time communication.

As I mentioned in the WG, a human-curated newsletter that reports on the recent/relevant activity in both in GraphQL JS, spec, RFC etc would be really helpful. Hard to get a pulse on what is important/relevant/related digging through GitHub. It should not generally include news/information on implementations other than GraphQL JS or third party libraries/tools unless they are relevant to the spec/RFC process.

I'd be curious if anyone on the Ember team has given a talk or if there are any blog posts on their process. At a minimum I think the easiest and appropriate thing would be to start posting these round-ups to the graphql.org blog. We can figure out a newsletter platform now or later. We could collaborate on content via a Google Doc or via a PR for graphql.org. A google doc is probably lower friction for drafting.

In summary I suggest:

- Identify who is willing to contribute (either content or editing)
- Limiting scope of communication to GraphQL as a technology - the spec, RFC, and relevant reference implementation(s), or projects at least tangentially related. 
- Coming up with a name for this type of communication (I think we should avoid the term "Weekly" in the name, even though weekly may be a good cadence, it may not be that regular)
- Posting the communication to the graphql.org blog
- At some point also creating a newsletter
- Start by creating a google doc shared with those willing to contribute

Any ideas for a name for the communication/newsletter?

**Comment by Zaba505** [comment]
Hi @IvanGoncharov and @craigsmitham, has there been any news on a "discussion space" (any medium for discussing WG ideas)? I've been trying to find any on my own and having no luck. I have a proposed idea for adding `import` statements to the GraphQL `TypeSystemDefinition` spec. I started by reading the README.md and it said to make a PR on the upcoming WG agenda file to discuss inclusions, proposals, etc. Unfortunately, at the moment, there isn't any new WG agenda file and it seems counter intuitive to make a PR for a meeting that already happened.

My solution to facilitate discussion for WG agenda proposals would be to have one single file (submittals.md, proposals.md, etc.) that PR's can be submitted to constantly. Then, before a specific WG meeting, the single files' merged PRs would be scanned and scheduled for that meeting. After the PRs are discussed and **concluded** they will be removed from the single file. This way there will always be a current and simple location for sending PRs, as well as, facilitates more frequent discussion through a (potential) higher volume of PRs.

Any thoughts on my solution would be greatly appreciated! Also, if there is any place I can run my `import`s idea by, that would be great!

**Comment by Zaba505** [comment]
@craigsmitham Thanks for the reply! Is there any timeline on when a process may be implemented? Also, is there a process for adding projects to the graphql org? I've been working on a GraphQL "compiler", [gqlc](https://github.com/Zaba505/gqlc) (really just a protoc knock off for the graphql idl). It's currently still in development but fast approaching a beta release and I was wondering if there is any proposal process for adding it to the graphql org code base?

**Comment by craigsmitham** [comment]
@Zaba505, to the best of my knowledge, the GraphQL org on GitHub is focused on projects that are related to the GraphQL specification or the GraphQL reference implementation. There are plenty of other community and industry projects focused on GraphQL that can be found at this page: https://graphql.github.io/code/. They all typically have their own Github org.
